runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java

```diff
@@ -307,28 +323,56 @@ private JPanel makeSkillPanel(HiscoreSkill skill)
 			// Cannot use SpriteManager as HiscorePlugin loads before a Client is available
 			directory += "/";
 		}
+		else if (bossKill)
+		{
+			directory = "bosses/";
+		}
 		else
 		{
 			directory += "_small/";
 		}
 
-		String skillIcon = directory + skillName + ".png";
+		String skillIcon = directory + skillName.replace("'", "").replace(":", "") + ".png";
 		log.debug("Loading skill icon from {}", skillIcon);
 
-		label.setIcon(new ImageIcon(ImageUtil.getResourceStreamFromClass(getClass(), skillIcon)));
-
-		boolean totalLabel = skill == HiscoreSkill.OVERALL || skill == null; //overall or combat
-		label.setIconTextGap(totalLabel ? 10 : 4);
+		JLabel labelIcon = new JLabel();
+		labelIcon.setIcon(new ImageIcon(ImageUtil.getResourceStreamFromClass(getClass(), skillIcon)));
 
+		boolean totalLabel = skill == OVERALL || skill == null; //overall or combat
+		labelText.setHorizontalAlignment(SwingConstants.RIGHT);
+		skillLabels.add(labelText);
+		skillIcons.add(labelIcon);
+		labelText.setIconTextGap(totalLabel ? 10 : 4);
 		JPanel skillPanel = new JPanel();
 		skillPanel.setBackground(ColorScheme.DARKER_GRAY_COLOR);
-		skillPanel.setBorder(new EmptyBorder(2, 0, 2, 0));
-		skillLabels.add(label);
-		skillPanel.add(skillLabels.get(skillLabels.size() - 1));
+
+		GridLayout grid = new GridLayout(1, 2);
+		skillPanel.setLayout(grid);
+		//Set different spacings based on skill or bosses
```

Comment:
why?

Reply:
To make the icons align properly regardless of the kc number when having bosses at 1kc vs bosses with 100k kc the layout got very wonky